src/main/java/com/fortify/plugin/jenkins/steps/FortifyCloudScanStep.java

```diff
@@ -28,13 +27,13 @@ public String getResolvedScanArgs(TaskListener listener) {
     protected String getCloudScanExecutable(Run<?, ?> build, FilePath workspace, Launcher launcher,
                                          TaskListener listener) throws InterruptedException, IOException {
         listener.getLogger().println("Checking for cloudscan executable");
-        return getExecutable("cloudscan" + (launcher.isUnix() ? "" : ".bat"), true, build, workspace, launcher,
-                listener, null);
+        return getExecutable("cloudscan" + (launcher.isUnix() ? "" : ".bat"), build, workspace,
+                listener, "FORTIFY_HOME");
     }
 
     protected String getScancentralExecutable(Run<?, ?> build, FilePath workspace, Launcher launcher,
                                             TaskListener listener) throws InterruptedException, IOException {
-        return getExecutable("scancentral" + (launcher.isUnix() ? "" : ".bat"), true, build, workspace, launcher,
-                    listener, null);
+        return getExecutable("scancentral" + (launcher.isUnix() ? "" : ".bat"), build, workspace,
+                listener, "FORTIFY_HOME");
```

Comment:
@vrulevskyi is FORTIFY_HOME set by Jenkins plugin somewhere or customer is suggested to set up it before running the agent? Usually FORTIFY_HOME is internal variable inside of or bat/bash executable files and it is calculated from file path. I don't know exactly how Jenkins plugin works and I am wondering if it would work or PATH variable would always be used for cloudscan/scancentral exec.

Reply:
FORTIFY_HOME is set by client in Jenkins global configuration. As can be seen from the above change, previously there was a  boolean flag which specified whether we had to look for an executable based on the value of FORTIFY_HOME env variable. It was always true for SCA and Apps executables. Now it's a name of env variable as the `getExecutable` method is adopted for other env variables, e. g. MAVEN_HOME or GRADLE_HOME. Also, if no FORTIFY_HOME is set then the executable will be searched in PATH according to current logic of `getExecutable` method.